liquibase-core/src/main/java/liquibase/change/core/CreateSequenceChange.java

```diff
@@ -89,7 +90,7 @@ public void setMinValue(BigInteger minValue) {
         this.minValue = minValue;
     }
 
-    @DatabaseChangeProperty(description = "Is the sequence generated in the requested order?")
```

Comment:
I appreciate that you removed the `?` from these. I understand the intent of the original description but it also kind of reads like, "hmm, will it or won't it? We'll let you try and find out!" Removing the `?` provides a more definitive statement.